Source/Scene/Scene.js

```diff
@@ -1668,7 +1668,7 @@ define([
             // TODO : Execute only opaque and translucent commands for now.
             var startPass = Pass.OPAQUE;
             var endPass = Pass.TRANSLUCENT;
-            for (var pass = startPass; pass < endPass; ++pass) {
+            for (var pass = startPass; pass <= endPass; ++pass) {
```

Comment:
I _think_ this is going to render the translucent pass twice in some cases.  Look farther down in this function to see when the translucent pass is done again (it depends on the OIT setting).

@bagnell...